modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/configuration/CPDefinitionLinkTypeSettingsImpl.java

```diff
@@ -29,22 +39,30 @@ public String[] getTypes() {
 	}
 
 	@Activate
-	protected void activate(BundleContext bundleContext) {
+	@Modified
+	protected void activate(
+		BundleContext bundleContext, Map<String, Object> properties) {
+
+		_cpDefinitionLinkTypeConfiguration =
```

Comment:
Please create a method `modified`, put the configuration initialization there and call the method here.